lib/Minz/ModelPdo.php

```diff
@@ -27,18 +26,21 @@ class Minz_ModelPdo {
 	protected $current_user;
 	protected $prefix;
 
-	public function dbType() {
-		return self::$sharedDbType;
-	}
-
 	/**
 	 * Créé la connexion à la base de données à l'aide des variables
 	 * HOST, BASE, USER et PASS définies dans le fichier de configuration
 	 */
-	public function __construct($currentUser = null) {
+	public function __construct($currentUser = null, $currentPrefix = null, $currentDb = null) {
```

Comment:
I am making a new PR, which will simply this part, together with other DB / PDO simplifications